packages/eslint-plugin/rules/jsdoc-third-person.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * ESLint rules: verb form.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+const { default: iterateJsdoc } = require( 'eslint-plugin-jsdoc/dist/iterateJsdoc' );
+
+/**
+ * Internal dependencies
+ */
+const { isDependencyBlock } = require( '../utils' );
+
+module.exports = iterateJsdoc( ( {
+	context,
+	jsdoc,
+	jsdocNode,
+} ) => {
+	if ( isDependencyBlock( jsdoc ) ) {
+		return;
+	}
+
+	if ( jsdoc.line === 0 ) {
+		return;
+	}
```

Comment:
Hmm, this actually raises a fair point. Wanting every word to be third-person makes a doc-block like this one, describing a constant/array, quite awkward:

```js
/**
 * Set of console logging types observed to protect against unexpected yet
 * handled (i.e. not catastrophic) errors or warnings. Each key corresponds
 * to the Puppeteer ConsoleMessage type, its value the corresponding function
 * on the console global object.
 *
 * @since 1.0.0
 *
 * @type {Object<string,string>}
 */
```

how would one change that to begin with a third-person verb? Doing so makes it less obvious, as the documented code in question isn't a function that _does_ something (where a verb makes sense), but a _constant_ that _is_ something (where a noun makes much more sense).

I suppose this rule should only apply for doc blocks not describing constant values. I'll see how obvious that is to implement—it could be tough as many functions are implemented as ES6 arrow functions with `const functionName = () => {};`.

Reply:
Sorted it by checking some other ESLint rule internals; I'll get this one added and fixed now 👍🏻

Reply:
@tofumatt Indeed, this rule actually should only apply to functions and class methods - not variable documentation or anything else. Can we make it so that this check is only made for those types of docblocks?